src/GitLink/Extensions/ProjectExtensions.cs

```diff
@@ -28,15 +28,22 @@ public static string GetProjectName(this Project project)
             return projectName ?? Path.GetFileName(project.FullPath);
         }
 
-        public static void CreateSrcSrv(this Project project, string rawUrl, string revision, Dictionary<string, string> paths, bool downloadWithPowershell)
+        public static void CreateSrcSrv(this Project project, string rawUrl, string revision, Dictionary<string, string> paths, bool downloadWithPowershell, Dictionary<string, string> vstsData = null)
```

Comment:
I am considering to move this into a SrcSrvContext class. Then we can introduce changes without breaking anything (we just have to add stuff to the SrcSrvContext class. Interested in adding this as well?